src/types/IJaaSMeetingProps.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import IJitsiMeetingOptions from "./IJitsiMeetingOptions";
+import { JaaSMeeting } from "..";
+
+/**
+ * The type of the React {@code Component} props of {@link JaaSMeeting}.
+ */
+export default interface IJaaSMeetingProps {
+  /**
+   * The id of the component
+   */
+  id: string;
+
+  /**
+   * The App ID that provides an isolated context and prefixes the room name
+   */
+  appId: string;
+
+  /**
+   * The optional arguments for the conference
+   */
+  options: IJitsiMeetingOptions;
+
+  /**
+   * The custom spinner to be displayed while the iframe is loading
+   */
+  spinner?: () => JSX.Element;
+
+  /**
+   * The external API reference for events and commands
+   */
+  onApiReady: (api: any) => object;
```

Comment:
Can you please add a types file for the external API, so this can be `JitsiMeetExternalAPIType` instead of `any` ?

Reply:
does this return `object`? or `void`?

Reply:
@horymury `void`, changed accordingly, ty!